### PR TITLE
chore: box refactor for status

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**

### CODE REVIEW

File: packages/odyssey-react/src/components/Status/Status.tsx [hunk 1]
```diff
@@ -63,10 +64,10 @@ let Status = forwardRef<HTMLDivElement, StatusProps>((props, ref) => {
   );
 
   return (
-    <div {...omitProps} className={styles.status} role="status" ref={ref}>
+    <Box {...omitProps} className={styles.status} role="status" ref={ref}>
       {labelElement}
       <span className={valueClass}>{descriptor}</span>
-    </div>
```
Comment (jeffbelser-okta) [inline packages/odyssey-react/src/components/Status/Status.tsx:69]: sanity checking if you disable the global styles for typography and reset in storybook here does this work just fine?

File: packages/odyssey-react/src/components/Status/Status.tsx [hunk 2]
```diff
@@ -63,10 +64,10 @@ let Status = forwardRef<HTMLDivElement, StatusProps>((props, ref) => {
   );
 
   return (
-    <div {...omitProps} className={styles.status} role="status" ref={ref}>
+    <Box {...omitProps} className={styles.status} role="status" ref={ref}>
```
Comment (jeffbelser-okta) [inline packages/odyssey-react/src/components/Status/Status.tsx:67]: need `as="div"` here

**Review by jeffbelser-okta — APPROVED** [review]
strong work 💪 quick sanity check inline

File: packages/odyssey-react/src/components/Status/Status.module.scss [hunk 3]
```diff
@@ -14,15 +14,13 @@ $status-indicator-size: 1em * 0.5;
 
 .label {
   display: block;
-  margin-block-end: $spacing-xs;
```
Comment (edburyenegren-okta) [inline packages/odyssey-react/src/components/Status/Status.module.scss:17]: Removing this causes a small regression. Add it back and good to go!

Comment (arnoldsandoval-okta) [reply packages/odyssey-react/src/components/Status/Status.module.scss]: Added back in.

File: packages/odyssey-react/src/components/Status/Status.tsx (hunk 1 above)
Comment (arnoldsandoval-okta) [reply packages/odyssey-react/src/components/Status/Status.tsx:69]: That is correct!

**Review by edburyenegren-okta — APPROVED** [review]
thanks AS!

**Pull request merged**